Cargo.toml

```diff
@@ -43,6 +43,7 @@ bench = false
 
 [features]
 default = ["prost-derive", "std"]
+prost-derive = ["dep:prost-derive"]
```

Comment:
Can we call this feature `derive`?

Reply:
That will break existing users of the feature, right?

Reply:
You're right...this is fine for now then...I didn't know cargo exposed this implicitly outside the crate itself.